refactor: add new eslint rules and apply changes

EncryptedDev opened the pull request.

Bas950 commented:
@EncryptedDev @ImRodry wen ist zis rdy?

EncryptedDev commented:
> @EncryptedDev @ImRodry wen ist zis rdy?

probably this weekend

ImRodry commented:
Yeah, will resolve the issues and hopefully finish it by this weekend, we also gotta compile all of them and I can’t do all that *cough* @EncryptedDev

EncryptedDev commented:
@Slowlife01 @Bas950 pls review

ImRodry commented:
I tried to keep the commit history clean but rhys added a merge commit
Either way we made a massCompiler script that checks for errors you’re welcome

ImRodry commented:
@Bas950 @Slowlife01 :)

TheDropX commented:
@Bas950

EncryptedDev commented:
@Bas950 @Slowlife01 now it is ready for review

down-bad commented on websites/W/Wolvesville/presence.ts:
```diff
@@ -322,23 +314,24 @@ presence.on("UpdateData", async () => {
                   "div.css-1dbjc4n.r-150rngu.r-eqz5dr.r-16y2uox.r-1wbh5a2.r-5oul0u.r-11yh6sk.r-1rnoaur.r-1sncvnh.r-13qz1uu"
                 )
-                ?.innerHTML.match(/text-decoration-line: line-through/g) || []
+                ?.textContent.match(/text-decoration-line: line-through/g) || []
```
why did the `innerHTML`s changed to `textContent`s? 
in this case it breaks the presence.

EncryptedDev replied:
textContent is better to use over innerHTML, maybe in this case it was not appropriate

Slowlife01 commented:
*On `no-one-time-vars` latest you can remove a few eslint-ignores*

ImRodry commented:
That’s what I did on the latest commit but it seems like I forgot to bump it

ImRodry commented:
@Bas950 @Slowlife01 pls

EncryptedDev committed:
refactor: add new eslint rules and apply changes

eslint rules

D

ya yeeet

fix: add setTrayTitle back

ok

refactor: more shit

yeppers

yeppers

trayTitle

yoppers

yeeters

yeet

boop

beep

wtf man

Revert "wtf man"

This reverts commit f2cae0c8a6352aff5aa87857b123a81a12f0a4be.

Revert "Revert "wtf man""

This reverts commit 0e31803fcdf3b52968d697174d666630afc3027f.

yoinkers

fix: presenceData and other

style: add two prettier rules

refactor: no more !== null

booooop

textContent => value (input elements)

more yeeters

style: prettier fixes

bump no-one-time-vars

fix: a, b, c, d, e, f websites

Update presence.ts

yeetus

feat: add metadata sorting script

refactor: sort metadatas

we made some script which we think works hopefully so!!!!!

apparently we fixed some stuff according to one smart little sexy beast

f

bumpy bumpy boopy i did an oopsie

eslint rules

D

ya yeeet

fix: add setTrayTitle back

ok

refactor: more shit

yeppers

yeppers

trayTitle

yoppers

yeeters

yeet

boop

beep

wtf man

Revert "wtf man"

This reverts commit f2cae0c8a6352aff5aa87857b123a81a12f0a4be.

Revert "Revert "wtf man""

This reverts commit 0e31803fcdf3b52968d697174d666630afc3027f.

yoinkers

fix: presenceData and other

style: add two prettier rules

refactor: no more !== null

booooop

textContent => value (input elements)

more yeeters

style: prettier fixes

bump no-one-time-vars

fix: a, b, c, d, e, f websites

Update presence.ts

yeetus

feat: add metadata sorting script

refactor: sort metadatas

we made some script which we think works hopefully so!!!!!

apparently we fixed some stuff according to one smart little sexy beast

Update metadata.json

fix more shiz

Update metadata.json

rename

rename

Small fixes for massCompile

Bump linting dependencies

Fix remaining errors

chore: bump the buggiest plugin ever

chore: fixed errors caused by the buggiest package

the buggiest package apparently now has less bugs raisedeyebrow

bumpy bumpy boopy i did an oopsie

Update websites/0-9/4chan/iframe.ts

fix: a buncha shit

refactor: yay

fix: bump more versions

chore: update schemas to 1.6

chore: rename xero

chore: rename xero

chore: bumped schema (#5093)

chore: add pr template (#5088)

feat(Prydwen Institute): Cover as largeImageKey (#5091)

* Update Prydwen Institude presence

* Small fix

* Fix typo

* Small fix

* Small fix

* Change metadata

* Update schema

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

feat(CSGO Stash): Cover as largeImagekey (#5095)

* feat(CSGO Stash): update presence

* Bump version

* Update schema

fix(Udemy): update title selector (#5096)

* updated course title selector

* restored missing anchor from udemy title selector

* Version bump Udemy presence

feat(Animu): Artwork as largeImageKey (#5092)

* Added simple artwork update

* Update metadata.json

* Revert "Update metadata.json"

This reverts commit 15021e76290f875320e1e25b7221435c5d8588b1.

* Update metadata.json

* Update websites/R/Rádio Animu/dist/metadata.json

feat(YouTube): Add thumbnails as image keys (#5099)

* feat(YouTube): Add thumbnails as image keys

* feat(YouTube): add thumbnail as a default off option

chore: bumped schema (#5093)

chore: add pr template (#5088)

feat(Prydwen Institute): Cover as largeImageKey (#5091)

* Update Prydwen Institude presence

* Small fix

* Fix typo

* Small fix

* Small fix

* Change metadata

* Update schema

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

feat(CSGO Stash): Cover as largeImagekey (#5095)

* feat(CSGO Stash): update presence

* Bump version

* Update schema

fix(Udemy): update title selector (#5096)

* updated course title selector

* restored missing anchor from udemy title selector

* Version bump Udemy presence

feat(Animu): Artwork as largeImageKey (#5092)

* Added simple artwork update

* Update metadata.json

* Revert "Update metadata.json"

This reverts commit 15021e76290f875320e1e25b7221435c5d8588b1.

* Update metadata.json

* Update websites/R/Rádio Animu/dist/metadata.json

feat(YouTube): Add thumbnails as image keys (#5099)

* feat(YouTube): Add thumbnails as image keys

* feat(YouTube): add thumbnail as a default off option

te

Update metadata.json

test

k

Update metadata.json

Co-Authored-By: Rhys <64903135+EncryptedDev@users.noreply.github.com>
Co-Authored-By: Bas950 <bas.van.zanten007@gmail.com>
Co-Authored-By: Slowlife <slowlife1165@gmail.com>
Co-Authored-By: Rodry <38259440+ImRodry@users.noreply.github.com>
